app/containers/MessageBox/index.tsx

```diff
@@ -1040,9 +1040,7 @@ class MessageBox extends Component<IMessageBoxProps, IMessageBoxState> {
 		const isAndroidTablet: Partial<IThemedTextInput> =
 			isTablet && isAndroid
 				? {
-						multiline: false,
-						onSubmitEditing: this.submit,
-						returnKeyType: 'send'
+						onSubmitEditing: this.submit
```

Comment:
If it behaves like the smartphone version, why do we need `isAndroidTablet`?

Reply:
Lmao true